atomic_reactor/plugins/pre_check_user_settings.py

```diff
@@ -81,6 +83,17 @@ def operator_bundle_from_scratch(self):
         ):
             raise ValueError(msg)
 
+    def multistage_docker_api_check(self):
+        """Check if multistage build isn't tried with docker_api"""
+        if self.workflow.builder.tasker.build_method != CONTAINER_DOCKERPY_BUILD_METHOD:
+            return
+
+        if len(self.workflow.builder.dockerfile_images.original_parents) > 1:
+            msg = "Multistage builds can't be build with docker_api," \
```

Comment:
```suggestion
            msg = "Multistage builds can't be built with docker_api, " \
```
It would be nice to check the full error message in unit tests

Reply:
done